contracts/release/release.go

```diff
@@ -135,10 +135,12 @@ func (r *ReleaseService) checkVersion() {
 
 	version, err := r.oracle.CurrentVersion(opts)
 	if err != nil {
-		if err == bind.ErrNoCode {
-			log.Debug("Release oracle not found", "contract", r.config.Oracle)
-		} else {
-			log.Error("Failed to retrieve current release", "err", err)
+		if err != les.ErrNoPeers {
+			if err == bind.ErrNoCode {
+				log.Debug("Release oracle not found", "contract", r.config.Oracle)
+			} else {
```

Comment:
Perhaps just add the new error check here:

```
else if err != les.ErrNoPeers {
  log.Error("Failed to retrieve current release", "err", err)
}
```

Reply:
done